applications/launchpad_v2/src/containers/MiningContainer/MiningViewActions/index.tsx

```diff
@@ -0,0 +1,26 @@
+import Button from '../../../components/Button'
+import t from '../../../locales'
+import SvgChart from '../../../styles/Icons/Chart'
+import SvgClock from '../../../styles/Icons/Clock'
+import SvgSetting2 from '../../../styles/Icons/Setting2'
+
+/**
+ * Renders set of links/actions in Mining dashboard
+ */
+const MiningViewActions = () => {
+  return (
+    <div>
+      <Button variant='text' leftIcon={<SvgClock />}>
```

Comment:
Font size incorrect, `<Button/>` component needs to be connected to `<Text/>` for correct font styling.

Reply:
I'm aware about this, I think about adding `<Text />` inside the `<Button />` component, so leaving as it is, and change in the button should fix this issue.